pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
 {% extends "pages/base_page.html" %}
 
-{% load wagtailcore_tags %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block meta %}
+  {{ block.super }}
+  <style>
```

Comment:
Nice, glad you used style tag. I realized my stub template had inline styles and I was thinking "I hope he doesn't actually do it this way" ;)